pkgs/by-name/ki/kirsch/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+  xorg,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "kirsch";
+  version = "0.2.6";
+
+  src = fetchzip {
+    url = "https://github.com/molarmanful/kirsch/releases/download/v${version}/kirsch-release_v${version}.zip";
+    hash = "sha256-4JUj69qXYOcjOO2HCD37c1QnCL1HlJkofoN3YghgYUc=";
+  };
```

Comment:
why not fetchFromGitHub?

Reply:
Commenting for the history.

With fetchzip I'm downloading an artifact that is built by a CI system, not the zip archive of the source code.

fetchFromGitHub is limited to downloading the source code.